inlanecruising_plugin/config/parameters.yaml

```diff
@@ -6,7 +6,7 @@ turn_downsample_ratio: 20 # Amount to downsample input lanelet centerline data o
 minimum_speed: 2.2352 # Minimum allowable speed in m/s
 speed_moving_average_window_size: 5 # Size of the window used in the moving average filter to smooth the output speeds
 curvature_moving_average_window_size: 9 # Size of the window used in the moving average filter to smooth the computed curvature
-back_distance: 20.0 # Number of meters behind the first maneuver that need to be included in points for curvature calculation
+back_distance: 30.0 # Number of meters behind the first maneuver that need to be included in points for curvature calculation
```

Comment:
Why was this changed? This seems totally unrelated to the feature added by this PR